__tests__/app.test.js

```diff
@@ -66,7 +66,7 @@ describe("04 GET /api/articles/:article_id", () => {
             topic: expect.any(String),
             created_at: expect.any(String),
             votes: expect.any(Number),
-            comment_count:expect.any(String)
+            comment_count: expect.any(String),
```

Comment:
comment_count needs to be a number. Have a look at how to cast one data type into another in an SQL query